cartoframes/auth/context.py

```diff
@@ -239,8 +240,7 @@ def read(self, table_name, limit=None, decode_geom=False, shared_user=None, retr
                 df = con.read('acadia_biodiversity')
         """
         # choose schema (default user - org or standalone - or shared)
-        schema = 'public' if not self.is_org else (
-            shared_user or self.creds.username())
+        schema = 'public' if not self.is_org else (shared_user or self.creds.username)
```

Comment:
I know this is a bit out of PR, but this is somewhat duplicated from `get_default_schema()` on line 212. Just wanted to flag it

Reply:
Yep, there is room from improvement, but as we are going to remove the context object very soon, I wouldn't spend too much time